src/GitLabApiClient/GitLabApiClient.csproj

```diff
@@ -8,12 +8,14 @@
     <FileVersion>1.0.0.0</FileVersion>
     <VersionSuffix>$(VersionSuffix)</VersionSuffix>
     <Copyright />
-    <Description>GitLabApiClient is a .NET rest client for GitLab API v4.</Description>
-    <Authors>nmklotas</Authors>
-    <PackageProjectUrl>https://github.com/nmklotas/GitLabApiClient</PackageProjectUrl>
+    <Description>This is a fork of GitLabApiClient (https://github.com/nmklotas/GitLabApiClient). I've added capabilities to upload files via the GitLab API.</Description>
```

Comment:
Can you leave the original csproj here and only add yourself as an author?

Reply:
Of course, sorry, this was not intended. Done!